mssql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -8,3 +8,34 @@ username=MSSQL_USERNAME
 password=MSSQL_PASSWORD
 schema=dbo
 outputSchema=[{"key":"ID","value":"string"},{"key":"LASTNAME","value":"string"}]
+
+datatypeColumns=(ID VARCHAR(100) PRIMARY KEY, COL1 TINYINT, COL2 SMALLINT, COL3 INT, COL4 BIGINT,\
+  COL5 BIT, COL6 DECIMAL(38, 0), COL7 DECIMAL(10, 4), COL8 DECIMAL(5, 0), COL9 SMALLMONEY, COL10 MONEY, \
+  COL11 NUMERIC(38, 0), COL12 NUMERIC(10, 4), COL13 FLOAT, COL14 REAL, COL15 DATE, COL20 TIME(0), COL21 CHAR(100), \
+  COL22 VARCHAR(100) , COL23 TEXT, COL24 NCHAR(100), COL25 NVARCHAR(100), COL26 NTEXT, COL27 BINARY(100), COL28 VARBINARY(100), \
+  COL30 HIERARCHYID, COL32 XML, COL33 GEOGRAPHY, COL34 GEOMETRY)
+datatypeColumnsList=(ID, COL1, COL2, COL3, COL4, COL5, COL6, COL7, COL8, COL9, COL10, COL11, COL12, \
+  COL13, COL14, COL15, COL20, COL21, COL22, COL23, COL24, COL25, COL26, COL27,  COL28, COL30, COL32, COL33, COL34)
+datatypeValues=VALUES ('USER1',1, 111, 1213232312, 1242343541515455, 1, 10, 1.1235, 1, 99999.0000, 10.5000, 1, 1.2346, 1.23, \
+  88888885248, '2023-01-01', '12:00:00', 'ABCDEF', 'ABC', 'Sample text', '� 好�?�?', \
+  '� 好�?�?', '� 好�?�?', 1, 0x12345, '/1/1/', '<name type="textbox">abc</name>', \
```

Comment:
Some characters are not getting loaded/copied correctly here (the ones marked with '?')